resources/knative-serving/templates/istio-knative-extras.yaml

```diff
@@ -0,0 +1,235 @@
+---
+# Source: istio/charts/gateways/templates/serviceaccount.yaml
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cluster-local-gateway-service-account
+  namespace: istio-system
+  labels:
+    app: cluster-local-gateway
+---
+
+
+---
+# Source: istio/templates/serviceaccount.yaml
+# NOTE: excluded because already part of the "istio" component
+
+---
+# Source: istio/templates/clusterrole.yaml
+# NOTE: excluded because already part of the "istio" component
+
+---
+# Source: istio/templates/clusterrolebinding.yaml
+# NOTE: excluded because already part of the "istio" component
+
+---
+# Source: istio/charts/gateways/templates/service.yaml
```

Comment:
Sorry Monsieur @antoineco we may also need to remove this <3

Reply:
I think those are important, we need to explain what was removed and why.

Reply:
You convinced me Monsieur, I removed these.